app/controllers/home_controller.rb

```diff
@@ -22,16 +23,41 @@ def set_host_info
 
   def set_all_markers
     @all_markers = []
-    Host.where.not(latitude: nil, longitude: nil).each do |h|
+    lat = [32.7490..34.7490]
+    lon = [-85.3880..-83.3880]
```

Comment:
Same comment as below:
What do these "magic numbers" represent?
May be good to put them in an a constant / variable with a descriptive name.

Also, if they are GA specific, maybe would make sense to have them as an env or config variable. So if people are rolling out in different states this is not hard-coded into the app

Reply:
yep good idea, that's ATL lat lon, ill make them env vars